### PR TITLE
[SDK-3277] Content changes to Angular Quickstart

### PR DESCRIPTION
**Pull request by frederikprijck**
A couple of adjustments to the Angular quickstart to be more aligned with the Style Guide.

### CODE REVIEW

File: articles/quickstart/spa/angular/interactive.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Install the Auth0 Angular SDK by running the following commands in your terminal
 npm install @auth0/auth0-angular
 ```
 
-The SDK exposes several types that help you integrate Auth0 with your Angular application idiomatically, including a module and an authentication service.
+The SDK exposes several types that help integrate Auth0 in an Angular application idiomatically, including a module and an authentication service.
 
 ## Register and Configure AuthModule {{{ data-action="code" data-code="app.module.ts#10:13" }}}
```
Comment (stevehobbsdev) [inline articles/quickstart/spa/angular/interactive.md]: Style guide: titles should be sentence case

```suggestion
## Register and configure AuthModule {{{ data-action="code" data-code="app.module.ts#10:13" }}}
```

File: articles/quickstart/spa/angular/interactive.md [hunk 2]
```diff
@@ -53,13 +53,13 @@ We use the [`forRoot()` pattern](https://angular.io/guide/singleton-services#the
 
 ## Add Login to Your Application {{{ data-action=code data-code="login-button.ts#11:13" }}}
```
Comment (stevehobbsdev) [inline articles/quickstart/spa/angular/interactive.md]:
```suggestion
## Add login to your application {{{ data-action=code data-code="login-button.ts#11:13" }}}
```

File: articles/quickstart/spa/angular/interactive.md [hunk 3]
```diff
@@ -53,13 +53,13 @@ We use the [`forRoot()` pattern](https://angular.io/guide/singleton-services#the
...
 
-The Auth0 Angular SDK gives you tools to quickly implement user authentication in your Angular application, such as creating a [login](/login) button using the `loginWithRedirect()` method from the `AuthService` service class. Executing `loginWithRedirect()` redirects your users to the Auth0 Universal Login Page, where Auth0 can authenticate them. Upon successful authentication, Auth0 will redirect your users back to your application.
+The Auth0 Angular SDK provides tools to quickly implement user authentication in an Angular application, such as creating a login button using the `loginWithRedirect()` method from the `AuthService` service class. Executing `loginWithRedirect()` redirects the users to the Auth0 Universal Login Page. Upon successful authentication, Auth0 will redirect the users back to the original application.
```
Comment (stevehobbsdev) [inline articles/quickstart/spa/angular/interactive.md]:
```suggestion
The Auth0 Angular SDK provides tools to quickly implement user authentication in an Angular application, such as creating a login button using the `loginWithRedirect()` method from the `AuthService` service class. Executing `loginWithRedirect()` redirects the users to the Auth0 Universal Login page. Upon successful authentication, Auth0 will redirect the users back to the original application.
```

File: articles/quickstart/spa/angular/interactive.md [hunk 4]
```diff
@@ -104,7 +107,9 @@ Still having issues? Check out our [documentation](https://auth0.com/docs) or vi
 
 ## Show User Profile Information {{{ data-action=code data-code="user-profile.ts" }}}
```
Comment (stevehobbsdev) [inline articles/quickstart/spa/angular/interactive.md]: Style guide: titles should be sentence case

```suggestion
## Show user profile information {{{ data-action=code data-code="user-profile.ts" }}}
```

**Review by stevehobbsdev — CHANGES_REQUESTED** [review]
👍🏻

**Review by stevehobbsdev — APPROVED** [review]

**Review by rkhoriander — APPROVED** [review]

**Pull request merged**